elements/rh-button/demo/rh-button.html

```diff
@@ -0,0 +1,54 @@
+<link rel="stylesheet" href="demo.css">
+<script type="module" src="rh-button.js"></script>
+
+<section>
+  <h2>Variants</h2>
+  <rh-button danger>Danger</rh-button>
+  <rh-button>Primary</rh-button>
+  <rh-button variant="link">Link</rh-button>
```

Comment:
Is link supposed to be used for links or to look like them? If the former, the button should be semantically a link, or at the very least, a role indicating it functions as one. If the latter, the user will be expecting it to behave as a link because of its visual styling.

Reply:
this problem is in pfv4

Reply:
apologies, please disregard my last comment in this thread, I was confused